### PR TITLE
Move exporting feature matrix example in docs

### PR DESCRIPTION
**Pull request by kmax12**
Doesn't need to be at the top level of the documentation and fits well within the deployment page

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Changelog
     * Changes
         * Moved dask, distributed imports (:pr:`634`)
     * Documentation Changes
-        * Miscellaneous changes (:pr:`641`)
```
Comment (gsheni) [inline docs/source/changelog.rst]: I think you might have to add another entry to get the changelog test to pass?

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ Changelog
...
+        * Miscellaneous changes (:pr:`641`, :pr:`658`)
         * Modified doc_string of top_n in encoding (:pr:`648`)
         * Hyperlinked ComposeML (:pr:`653`)
     * Testing Changes
```
Comment (gsheni) [inline docs/source/changelog.rst]: Missing user name 4 lines below

**Comment by rwedge** [comment]
Can you add yourself to the list of contributors?

File: docs/source/guides/deployment.rst [hunk 3]
```diff
@@ -68,3 +68,29 @@ After we load the features back in, we can calculate the feature matrix.
 As you can see above, we have the exact same features as before, but calculated on using our test data.
```
Comment (rwedge) [inline docs/source/guides/deployment.rst]: "calculated on using our test data" sounds awkward

**Pull request merged**